owslib/wps.py

```diff
@@ -851,7 +851,7 @@ def isSucceded(self):
     def isNotComplete(self):
         return not self.isComplete()
 
-    def getOutput(self, filepath=None):
+    def getOutput(self, filepath=None, identifier=None):
         """
         Method to write the outputs of a WPS process to a file:
         either retrieves the referenced files from the server, or writes out the content of response embedded output.
```

Comment:
document identifier parameter in docstring.

Reply:
document that it will default to first output for backward compatibility.

Reply:
done.